src/cr/cube/slices.py

```diff
@@ -282,6 +283,11 @@ def table_margin(self):
         return np.sum(self._base_counts)
 
 
+# Used to represent the non-existent dimension in case of 1D vectors (that need to be
+# accessed as slices, to support cr.exporter).
+_PlaceholderElement = namedtuple("_PlaceholderElement", "label, is_hidden")
```

Comment:
Should this be `"label", "is_hidden"`? looks like you missed the interior set of double-quotes.

Reply:
It's funny for me as well, but it works like this :|

Reply:
Example from [this](https://docs.python.org/2/library/collections.html#collections.namedtuple) page:

```
EmployeeRecord = namedtuple('EmployeeRecord', 'name, age, title, department, paygrade')
```